content/concepts/tokens/nftoken-authorized-minting.md

```diff
@@ -0,0 +1,48 @@
+---
+html: nftoken-authorized-minting.html
+parent: non-fungible-tokens.html
+blurb: You can assign another account to mint NFTs in your stead.
+labels:
+ - Non-fungible Tokens, NFTs
+---
+
+# Authorizing Another Account to Mint Your NFTs
+
+Each account can have 0 or 1 authorized minter that can mint NFTs on its behalf. Combined with the brokered sales, creators would have the option of allowing other accounts to mint and sell NFTs for them, so that they can focus on making more NFTs. 
```

Comment:
I'm not sure how brokered sales fit into this topic?  The following sentence works a bit better for me:

"By authorizing a minter, a creator can allow a different account to mint and sell NFTs for them, which allows the creator to focus on making more NFTs."

Just for your consideration...